package.json

```diff
@@ -8,10 +8,11 @@
         "start": "node --optimize_for_size --gc_interval=100 --max_old_space_size=920 dist/bot.js",
         "deploy": "node dist/deploy.js",
         "build": "swc ./src -d dist --delete-dir-on-start",
+        "get-luau": "node get-luau.js",
         "dev": "cross-env-shell NODE_ENV=development \"pnpm build && pnpm run deploy && pnpm start\"",
         "type-check": "tsc",
         "lint": "eslint \"src/**/*.ts\"",
-        "prettier": "prettier --write \"src/**/*.ts\""
+        "prettier": "prettier --write \"{get-luau,src/**/*}.{ts,js}\""
```

Comment:
the get-luau file won't be checked by prettier since it's in the prettier ignore

Reply:
The .prettierignore file has been edited